components/avatar/avatar.css

```diff
@@ -0,0 +1,37 @@
+@import url('./main.css');
+.avatar-sm{
+    height: 2.5rem;
+    width: 2.5rem;
+    object-fit: cover;
+    background-color: var(--Elevation-2);
+    border-radius: .75rem;
+    position: relative;
+}
+.avatar{
+    height: 3.5rem;
+    width: 3.5rem;
+    object-fit: cover;
+    border-radius: .75rem;
+    position: relative;
+}
+.avatar-md{
+    height: 4.5rem;
+    width: 4.5rem;
+    object-fit: cover;
+    border-radius: .75rem;
+    position: relative;
```

Comment:
Since these 3 styles are being repeated for every avatar instead of writing it multiple times, you can write it once by combining all selectors using a comma.